packages/payment-processor/test/payment/erc777-stream.test.ts

```diff
@@ -239,4 +241,53 @@ describe('erc777-stream', () => {
       expect(paymentFlowRate).toBe('0');
     });
   });
+
+  describe('makeErc777OneOffPayment', () => {
+    it('Should perform a payment', async () => {
+      // initialize the superfluid framework...put custom and web3 only bc we are using ganache locally
+      const sf = await Framework.create({
+        networkName: 'custom',
+        provider,
+        dataMode: 'WEB3_ONLY',
+        resolverAddress: RESOLVER_ADDRESS,
+        protocolReleaseVersion: 'test',
+      });
+
+      // use the framework to get the SuperToken
+      const daix = await sf.loadSuperToken('fDAIx');
+
+      const senderBalanceBefore = await daix.balanceOf({
+        account: wallet.address,
+        providerOrSigner: provider,
+      });
+      const recipientBalanceBefore = await daix.balanceOf({
+        account: paymentAddress,
+        providerOrSigner: provider,
+      });
+
+      // Perform the payment
+      const tx = await makeErc777OneOffPayment(
+        validRequest,
+        BigNumber.from(oneOffPaymentAmount),
+        wallet,
+      );
+      await tx.wait(1);
+
+      const senderBalanceAfter = await daix.balanceOf({
+        account: wallet.address,
+        providerOrSigner: provider,
+      });
+      const recipientBalanceAfter = await daix.balanceOf({
+        account: paymentAddress,
+        providerOrSigner: provider,
+      });
+
+      expect(BigNumber.from(senderBalanceBefore)).toEqual(
+        BigNumber.from(senderBalanceAfter).add(oneOffPaymentAmount),
```

Comment:
I don't understand, why does it work? Shouldn't it be `.sub(` the amount?

Reply:
I guess @yomarion means balanceAfter = balanceBefore - sentAmount